store-tests

```diff
@@ -195,6 +209,10 @@ def insert_test(cursor, run_id, testname, failed, retry_reason, seconds):
                    (testname, retry_reason, int(failed), run_id, int(seconds)))
 
 
+def insert_coverage(cursor, run_id, coverage):
+    cursor.execute("INSERT INTO TestCoverage VALUES (?, ?)", (coverage, run_id))
```

Comment:
Hmm, but we only ever want to store *one* value in that table, no? (the latest one). So doesn't this need some `DELETE FROM` first, or some `UPDATE`?

Reply:
No, I want to be able to plot how the coverage behaves over time. So storing historical values makes sense.
For plotting itself we want to probably take the last/highest value for each day. Also this would store retries coverage as well, which is possibly pointless but it might show some interesting data over time.

Reply:
ack, thanks!